src/agents/coordinator.ts

```diff
@@ -24,12 +24,23 @@ export interface CoordinatorRitual {
 export type DkgParticipant = {
   provider: string;
   aggregated: boolean;
+  // TODO: How do I get the transcript from the Coordinator?
+  transcript: Transcript;
```

Comment:
I assume this TODO is no longer relevant? Yep, this is how you get the transcript

Reply:
:heavy_check_mark: